### DISCUSSION TITLE
spring custom validations in DGS

### DISCUSSION THREAD

**Discussion by ilterpehlivan** [Netflix/dgs-framework#1151] [Q&A]
Hi,

Is there any way to combine spring validations with DGS framework ? Here is the spring documentation related to the functionality I am seeking for; https://docs.spring.io/spring-framework/docs/current/reference/html/web.html#mvc-config-validation <br>

Thanks
Ilter

**Comment by srinivasankavitha** [comment]
Have you tried it using the described mechanism with DGS annotated classes? We don't do anything special to support this. That said, I would think it should work out of the box.
